extensions/security-jpa/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```
@@ -0,0 +1,12 @@
+---
+name: "Quarkus Security JPA"
```

```suggestion
name: "Security JPA"
```
Given what I see on `code.quarkus.io`, it shouldn't have Quarkus in the name.